sql/triggers.sql

```diff
@@ -80,10 +80,19 @@ CREATE OR REPLACE FUNCTION pg2kafka.setup(table_name_ref regclass, external_id_n
 LANGUAGE plpgsql
 AS $_$
 DECLARE
+  existing_id varchar;
   trigger_name varchar;
   lock_query varchar;
   trigger_query varchar;
 BEGIN
+  SELECT pg2kafka.external_id_relations.external_id INTO existing_id
+  FROM pg2kafka.external_id_relations
+  WHERE pg2kafka.external_id_relations.table_name = table_name_ref::varchar;
+
+  IF existing_id != '' THEN
+    RETURN;
```

Comment:
Do you think we should log something instead of silently ignoring the fact that someone tried to run this twice?